00-intro-carto/exercises/intro.md

```diff
@@ -0,0 +1,63 @@
+# CARTO BUILDER, Dashboards, Dataset View and Public Profile Introduction
+
+## Contents
+
+0. [Resources](#resources)
+1. [Maps Dashboard](#maps)
+2. [Datasets Dashboard](#datasets)
+3. [Dataset View](#dataset-view)
+4. [BUILDER](#builder)
+5. [Public Profile](#profile)
+
+### 0. Resources <a name="resources"></a>
+
+* [CARTO Documentation](https://carto.com/docs/)
+* [LEARN Guides](https://carto.com/learn/guides)
+
+### 1. Maps Dashboard <a name="maps"></a>
+
+* URL: `https://{{USER_NAME}}.carto.com/dashboard/dataset`
+
+![maps-dashboard](imgs/maps-dashboard.png)
+
+* CARTO Maps Dashboard shows BUILDER maps ordered by `date` (default), `visits` or `likes`, users can `SEARCH` and switch to the maps with `likes`. 
+* When a map is selected, users can duplicate, delete or lock it, or change its privacy settings.
+* Also, users can change maps privacy settings and add tags, description, likes and description to their maps. New maps can be imported clicking on `NEW MAP` or dragging and dropping a .carto file. 
+* Finally, users can go to their profile settings clicking on their profile picture, see their notifications and go to their `DATASETS`, `LOCKED MAPS` or `DATASETS` from the dropdown menu appearing after clicking on `Maps`.
+
+### 2. Datasets Dashboard <a name="datasets"></a>
+
+* URL: `https://{{USER_NAME}}.carto.com/dashboard`
+
+![datasets-dashboard](imgs/datasets-dashboard.png)
+
+* CARTO Datasets Dashboard shows BUILDER datasets ordered by `date` (default), `visits`, `likes` or `size`, users can `SEARCH`, switch to the datasets with `likes` and go to CARTO `DATA LIBRARY`. 
+* When a dataset is selected, users can duplicate, delete or lock it, change its privacy settings or create a map.
+* Also, users can change datasets privacy settings and add tags, description, likes and description to their maps. 
+* New datasets can be imported clicking on `NEW DATASET` or dragging and dropping a supported file. 
+* Finally, users can go to their profile settings clicking on their profile picture, see their notifications and go to their `MAPS`, `LOCKED MAPS` or `DATASETS` from the dropdown menu appearing after clicking on `Datasets`.
+
+### 3. Dataset View <a name="dataset-view"></a>
+
+* URL: `https://{{USER_NAME}}.carto.com/dataset/{{TABLE_NAME}}`
+
+![dataset-view](imgs/dataset-view.png)
+
+* CARTO users can rename, duplicate, edit metadata, lock or delete it.
+* They can also change it profile settings.
+* Also, it is possible to `ADD ROW`s, `COLUMN`s and `EXPORT` it.
```

Comment:
"They can also change it profile settings" sounds weird to me. What are you trying to say? Try to avoid starting sentences with Also, or Finally. Use active voice or bullets.